include/slate/internal/MatrixStorage.hh

```diff
@@ -346,11 +346,14 @@ public:
     scalar_t* allocWorkspaceBuffer(int device);
     void      releaseWorkspaceBuffer(scalar_t* data, int device);
 
+private:
+    // Iterator routines should only be called with in a Tiles Map LockGuard
+    // Otherwise, there may be race conditions with the returned iterator
```

Comment:
"should be called only within a ..." (move "only", contract "with in").
Per style guide, sentences in comments should end with period (2x here).